install/kubernetes/helm/istio/charts/galley/templates/validatingwebhook.yaml

```diff
@@ -33,6 +33,14 @@ webhooks:
         - httpapispecbindings
         - quotaspecs
         - quotaspecbindings
+      - operations:
```

Comment:
We can validate all kinds/versions once RBAC is in its own group (see below). The version/kind specific qualification was only required for `config.istio.io` which is split across mixer and pilot validation.

```yaml
- operations:
  - CREATE
  - UPDATE
  apiGroups:
  - rbac.istio.io
  apiVersions:
  - "*"
  resources:
  - "*"
```